<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SellerSalary</class>
 <widget class="QDialog" name="SellerSalary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QLabel" name="endD">
         <property name="text">
          <string>Конечная дата</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="beginD">
         <property name="text">
          <string>Начальная дата</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QDateEdit" name="dateEnd"/>
       </item>
       <item row="0" column="0">
        <widget class="QDateEdit" name="dateBegin"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="result">
     <property name="minimumSize">
      <size>
       <width>141</width>
       <height>41</height>
      </size>
     </property>
     <property name="text">
      <string>Получить результат</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
